Sources/MongoSwift/ReadPreference.swift

```diff
@@ -96,6 +96,11 @@ public final class ReadPreference {
         self._readPreference = mongoc_read_prefs_new(mode.readMode)
     }
 
+    /// Initialize a new empty `ReadPreference` with the server's default.
+    public init() {
+        self._readPreference = mongoc_read_prefs_new(MONGOC_READ_PRIMARY)
```

Comment:
to reduce our libmongoc exposure, it might be better if we routed this through an existing initiailzer.
e.g.
```swift
self.init(.primary)
```